packages/components/psammead-navigation/README.md

```diff
@@ -146,26 +147,30 @@ import { latin } from '@bbc/gel-foundations/scripts';
 
 ```jsx
 import React from 'react';
-import { Dropdown, DropdownUl, DropdownLi } from '@bbc/psammead-navigation/dropdown';
+import {
+  Dropdown,
+  DropdownUl,
+  DropdownLi,
+} from '@bbc/psammead-navigation/dropdown';
 import { latin } from '@bbc/gel-foundations/scripts';
 
 <Dropdown>
   <DropdownUl>
     <DropdownLi
-        script={latin}
-        service="news"
-        key="Home"
-        url="/"
-        active
-        currentPageText="Current page"
+      script={latin}
+      service="news"
+      key="Home"
+      url="/"
+      active
+      currentPageText="Current page"
     >
       Home
     </DropdownLi>
     <DropdownLi script={latin} service="news" key="Sport" url="/sport">
       Sport
     </DropdownLi>
   </DropdownUl>
-</Dropdown>
+</Dropdown>;
```

Comment:
```suggestion
</Dropdown>
```

Reply:
Hmm... Prettier decided to add all of these in